### PR TITLE
z80ne: Added software lists for disks and tapes

### PR DESCRIPTION
**Pull request by rdroush**

### CODE REVIEW

File: src/mame/drivers/z80ne.cpp [hunk 1]
```diff
@@ -221,7 +222,7 @@ PORT_BIT( 0x01, IP_ACTIVE_LOW, IPT_KEYBOARD ) PORT_NAME("LX.384 Reset")  PORT_CO
 
 /* Settings */
 PORT_START("LX.385")
-PORT_CONFNAME(0x07, 0x01    , "LX.385 Cassette: P1,P3 Data Rate")
+PORT_CONFNAME(0x07, 0x04, "LX.385 Cassette: P1,P3 Data Rate")
```
Comment (cuavas) [inline src/mame/drivers/z80ne.cpp]: Why the change in default value?

**Comment by rb6502** [comment]
@rdroush Any comment on the weird change there?

**Comment by Robbbert** [comment]
If you wish to send the software to [email](http://wiki.mamedev.org/images/f/ff/Submit-email.png) with links to where this data is located, then I'll be happy to merge this PR.

**Comment by rb6502** [comment]
Since this is so old and they never replied to why they made the change Vas noted, I'm going to give this another 24 hours and then close it.

**Comment by Robbbert** [comment]
They have cassettes in the software list, which I presume are recorded at 1200 baud. That would be the reason for the change.

**Comment by Robbbert** [comment]
Well unfortunately @rdroush has not responded with supply of required software, or anything else.

So, closing.

**Comment by LnmVolbo** [comment]
Some stuff for this was posted a few days ago here : http://www.retroroms.info/forum/topic/MAME-0-184-SL-CHD-sooner-requests.htm#1245 (courtesy of **MdGheist**, but he didn't bother to state where the files were obtained)

That site requires free registration to download attachments, so if that's a problem, here's a sendspace link: https://www.sendspace.com/filegroup/mo0rncd2sP7eSAcjELcrEHwxNDBKtNcM

Since this roundabout way of submitting the files may not fulfill your validation requirements, it is understandable if you cannot merge this PR unless you hear from **rdoush** directly.

**Comment by Robbbert** [comment]
@LnmVolbo : just noticed your comment just now, and the sendspace link is already dead. However I was able to find the software elsewhere, so this can now be safely merged.
